d2-milestone/src/components/question/CreateQn.jsx

```diff
@@ -24,39 +25,46 @@ function CreateQn({handleClose, addQuestion}) {
         navigate('/')
       } 
     )
-    .catch(err => console.log(err))
+    .catch(e => {
+        if (e.response && e.response.status === 400) {
+                setError(e.response.data.error)
+                // console.log("error is:", error)
+            }
+            console.error('Error updating question:', e);
+    })
   }
 
   return (
     <div className='d-flex bg-primary justify-content-center align-items-center'>
         <div className="w-100 bg-white p-3">
             <form onSubmit={Submit}>
+                {error && <div className="alert alert-danger">{error}</div>}
                 <div className="mb-2">
                     <label htmlFor="">Category</label>
-                    <input type="text" placeholder='Data Structures' className='form-control' required
```

Comment:
Alrighty, added the `name={"complexity"}` 

I omitted the `required` tag because the way it shows the error msg is the default HTML5 style (first pic) but I wanted to show customised error msg (second pic) so I removed the `required` tag
<img width="300" alt="image" src="https://github.com/user-attachments/assets/74c4991b-5020-4702-a023-9bb51e87f6a9">
<img width="300" alt="image" src="https://github.com/user-attachments/assets/e1dcfcd3-8de5-434c-b281-bdffdeeedaa5">